src/modules/SuSEFirewallCMDLine.rb

```diff
@@ -1599,6 +1634,40 @@ def Run
       nil
     end
 
+  private
+    # Returns true if FirewallD is the running backend
+    def firewalld?
+      return true if SuSEFirewall.is_a?(Yast::SuSEFirewalldClass)
```

Comment:
well, problem is that it is not obvious that SuSEFirewalldClass is pulled by SuSEFirewall.rb, so I will prefer to have here require of SuSEFirewalld file ( ruby ensure it is not loaded twice, no worry ). It is just if we decide in future that SuSEFirewall do not require SuSEFirewalldClass it won't break here.